tutorial-compiler/index.asciidoc

```diff
@@ -0,0 +1,109 @@
+= Tutorial Compiler 
+==== 
+In this scenario you will use the tutorial-compiler to generate the "my-thai-star" tutorial.
+
+The first part will be to setup your workspace.
+
+Afterwards we will show you, how to write your own tutorials using an asciidoc.
+
+Requirements: 
+ * github-account
+
+You can find the documentation here:
+
+https://github.com/devonfw-forge/tutorial-compiler/wiki
+
+====
+
+====
+# Setup your workspace
+Let's start!
+
+First of all, go to the tutorial-compiler repository on github:
+
+https://github.com/devonfw-tutorials/tutorial-compiler
+
+[step]
+--
+nextKatacodaStep("Fork the 'tutorial-compiler'-repository", [{ "image": "images/fork.jpg" }])
+--
+
+Fork the repository by clicking on the fork-button.
+====
+
+====
+Go to your repositories, click on 'tutorial-compiler' and copy the repository Url as shown in image below.
```

Comment:
Sometimes you use Url and sometimes URL. I would suggest to write URL everywhere, because it is an abbreviation.